stdlib/email/src/main/ballerina/src/email/service_endpoint.bal

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) 2020 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerinax/java;
+
+public function initSmtpClientEndpoint(SmtpClient clientEndpoint, SmtpConfig config) returns error? = @java:Method {
+    name : "initClientEndpoint",
+    class : "org.ballerinalang.stdlib.email.client.SmtpClient"
+} external;
+
+public function send(SmtpClient clientEndpoint, Email email) returns error? = @java:Method {
+    name : "sendMessage",
+    class : "org.ballerinalang.stdlib.email.client.SmtpClient"
+} external;
+
+
+public function initPopClientEndpoint(PopClient clientEndpoint, PopConfig config, boolean isPop) returns error? = @java:Method {
+    name : "initClientEndpoint",
+    class : "org.ballerinalang.stdlib.email.client.PopClient"
+} external;
+
+public function popRead(PopClient clientEndpoint, Filter message, boolean isPop) returns Email|error? = @java:Method {
+    name : "readMessage",
+    class : "org.ballerinalang.stdlib.email.client.PopClient"
+} external;
+
+
```

Comment:
```suggestion
```

Reply:
Purposely separated with 2 lines to separate between protocols.